pkg/pool/dummypool.go

```diff
@@ -0,0 +1,98 @@
+package pool
+
+import (
+	"sync"
+
+	"github.com/pg-sharding/spqr/pkg/models/kr"
+	"github.com/pg-sharding/spqr/pkg/models/spqrerror"
+	protos "github.com/pg-sharding/spqr/pkg/protos"
+	"github.com/pg-sharding/spqr/pkg/shard"
+)
+
+type DummyPool struct {
```

Comment:
I do dislike Dummm* naming. I do not understand why ConnectionKeeper is bad.

Reply:
ConnectionKeeper is a good naming, I combine CoordPool and ConnectionKeeperData here